### DISCUSSION TITLE
Plot performance

### DISCUSSION THREAD

**Discussion by ebeng** [Chia-Network/chia-blockchain#6682] [Q&A]
I have a few Dell r420 servers where i do the plotting. 
However when I check the htop, I dont see all the CPUs beging used. Only just 1 out of 12 threads. Any idea how I can optimize?

**Comment by cccat6** [comment]
The official plotter can only utilize multi-threads in phase 1. The remaining 3 phases are all single-threaded.
You can parallel more plotting tasks or try https://github.com/madMAx43v3r/chia-plotter

**Reply by ebeng** [reply]
Ahhh... that makes sense...
I will try on one of the servers as he is explaining, I do have the same cpus and now im getting plots with 10+ hours...
